libs/blocks/merch/merch.js

```diff
@@ -349,7 +377,13 @@ export async function initService(force = false) {
   const { env, commerce = {}, locale } = getConfig();
   commerce.priceLiteralsPromise = fetchLiterals(PRICE_LITERALS_URL);
   initService.promise = initService.promise ?? polyfills().then(async () => {
-    const commerceLib = await import('../../deps/commerce.js');
+    const { hostname, searchParams } = new URL(window.location.href);
```

Comment:
Couldn't the method itself get the `hostname` and `searchParams`? What's the value of sending these from here?

Reply:
I kept `getMasBase` as a pure function so that it is easier to test it.
Since we need to check whether `maslibs` param is present here and `getMasBase` only needs `hostname ` and `maslibs` parameters, I limited it to those params.

Reply:
we will address this in a different PR.